corehq/apps/reports/formdetails/readable.py

```diff
@@ -53,6 +54,11 @@ def get_questions(domain, app_id, xmlns):
     try:
         xform = app.get_xform_by_xmlns(xmlns)
     except ResourceNotFound:
+        _soft_assert = soft_assert('@'.join(('devops+soltech', 'dimagi.com')))
```

Comment:
Should we not read this email address from a file so that this error will go to local hosting teams' own members?